.github/workflows/ci.yaml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+jobs:
+  python-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - name: Install viewerutil
+        run: pip install .[dev]
+
+      - name: Run python tests
+        run: pytest
+
+  sign-windows:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Test sign-windows (Multiple files)
+        uses: ./sign-windows
+        with:
+          vault-uri: "${{ secrets.AZURE_KEY_VAULT_URI }}"
+          cert-name: "${{ secrets.AZURE_CERT_NAME }}"
+          client-id: "${{ secrets.AZURE_CLIENT_ID }}"
+          client-secret: "${{ secrets.AZURE_CLIENT_SECRET }}"
+          tenant-id: "${{ secrets.AZURE_TENANT_ID }}"
```

Comment:
These secrets are stored for the `secondlife/viewer` and `secondlife/viewer-private` repos, not for `viewer-build-util`. Or did you store them here too?

Reply:
I've set this repository up with its own AZURE_ secrets in order to exercise the action in an end-to-end fashion.